pages/team.md

```diff
@@ -46,7 +46,7 @@ Kay is a technical writer at Inrupt and a newcomer to Solid.
 
 ## Kyra Assaad
 **Creator**
-Kyra Assaad is the Product Designer at Inrupt. She works to ensure that the Solid user experience is accessible and useable for all. She serves as one of the chairs of (and helped form) the Solid Diversity, Equity, and Inclusion (DEI) team. Whose goal is to increase voices in the Solid community by inviting individuals from all underrepresented groups..
+Kyra Assaad is the Product Designer at Inrupt. She works to ensure that the Solid user experience is accessible and useable for all. She helped form and serves as one of the chairs of the Solid Diversity, Equity, and Inclusion (DEI) team, in order to increase representation and participation of women and people of color in the Solid community.
```

Comment:
```suggestion
Kyra Assaad is the Product Designer at Inrupt. She works to ensure that the Solid user experience is accessible and useable for all. She helped form and now serves as one of the chairs of the Solid Diversity, Equity, and Inclusion (DEI) team, whose goal is to increase representation and participation of women and people of color in the Solid community.
```


Otherwise, it seems like your serving as a DEI chair increases representation and participation of women and people of color, which statistically is true but am guessing that's not the intent of that sentence 😄  (but maybe it is 😜 )

Reply:
Yeah, it's tricky- I'm trying to explain why I personally helped form/join the DEI team, as opposed to what the DEI team is about. Let me take another crack at it

Reply:
Maybe "participation of women,  people of color and other under-represented groups".

Reply:
I like the way you've rephrased it and withdraw my suggestion.